org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/wizard/project/ProjectPreviewWizardPage.java

```diff
@@ -393,7 +415,14 @@ private void clearTree() {
 
                 @Override
                 public void run() {
-                    ProjectPreviewWizardPage.this.projectPreviewTree.removeAll();
+                    try {
+                        if (!ProjectPreviewWizardPage.this.projectPreviewTree.isDisposed()) {
+                            ProjectPreviewWizardPage.this.projectPreviewTree.removeAll();
+                        }
+                    } catch (SWTException e) {
```

Comment:
If we don't catch it, then there is no way to log it, which means that there is no way to test for it. In an earlier conversation between @donat and I, it was suggested that this fix implement the logging.